react/components/Table/SimpleTable.js

```diff
@@ -72,9 +102,13 @@ class SimpleTable extends Component {
       updateTableKey,
       rowHeight,
       fullWidth,
+      lineActions,
       loading,
     } = this.props
     const { hoverRowIndex } = this.state
+
+    if (lineActions)
+      schema.properties = this.addLineActionsToSchema(schema, lineActions)
```

Comment:
Not sure if this mutation might cause an issue 🤔

Reply:
This would only add one column with the prop `lineActions` is passed. What do you think it could cause?

Reply:
seems ok now :)